### PR TITLE
Adjust colors and few improvement

### PR DESCRIPTION
**Pull request by YusukeHosonuma**
- [x] Adjust color of default theme
- [x] Add new-line when output log files (for more readable)
- [x] Refactor - rename
- [x] Update README.md

Looks like:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -376,18 +376,42 @@ You can customize terminal ANSI colors by `Debug.Option.theme` property like as
...
-    nilLiteral: { $0.yellow() },
-    boolLiteral: { $0.yellow() },
-    stringLiteral: { $0.blue() },
-    numberLiteral: { $0.cyan() }
+    type: { $0.green().bold() },
+    nil: { $0.yellow() },
+    bool: { $0.yellow() },
+    string: { $0.blue() },
+    number: { $0.cyan() },
+    url: { $0.underline() }
```
Comment (YusukeHosonuma) [inline README.md:384]: Renamed to more simply and intuitive.

File: README.md [hunk 2]
```diff
@@ -376,18 +376,42 @@ You can customize terminal ANSI colors by `Debug.Option.theme` property like as
...
 )
 
 Debug.sharedOption = Debug.Option(theme: theme)
 ```
 
 ANSI colors can be easily specified using [ColorizeSwift](https://github.com/mtynior/ColorizeSwift).
 
+#### Did you create a beautiful theme?
```
Comment (YusukeHosonuma) [inline README.md:392]: Welcome to contributions!

Comment (sahara-ooga) [reply README.md:392]: A bold challenge :-)

**Review by sahara-ooga — APPROVED** [review]
I think the changes makes the coloring API more intuitive.
Raising color themes is an interesting move👍🏻

**Pull request merged**